src/containerapp/azext_containerapp/_params.py

```diff
@@ -142,6 +146,13 @@ def load_arguments(self, _):
     with self.argument_context('containerapp env dapr-component init') as c:
         c.argument('statestore', help="The state store component and dev service to create.")
         c.argument('pubsub', help="The pubsub component and dev service to create.")
+    
+    with self.argument_context('containerapp env identity', is_preview=True) as c:
```

Comment:
Please add examples for all new parameters in` _help.py` for command `containerapp env identity`.